docs/release.rst

```diff
@@ -321,10 +321,83 @@ There are two generally multi-week phases during which the upcoming Fedora relea
 
 During these periods of time only accepted freeze exceptions and blocker fixes are allowed to reach the stable repository.
 
-So don't merge any PRs to the fedora-<version> upstream branch during a Fedora freeze that don't fix a freeze exception or a release blocker.
+We can't realistically enforce this policy on the main branch (effectively freezing it as well) we handle Fedora releases
+during the freeze period via a patch build.
+
+Unlike with a patch build where for each anaconda package release a new tarball is create, for a patch build one more patches
+are applied on top of the existing tarball at build time. These patches contain the approved blocker fixes and freeze exceptions.
+
+A patch build by example in a few easy steps:
+
+1. identify which PRs fall under the approved blocker bug or freeze exception & make sure each commit has
+
+::
+
+    Resolves: rhbz#<bug_id> (in this example we will use 1234)
+
+2. cherry pick the commit using the ``git cherry-pick command``, resulting in one patch file per commit using the ``git format-patch`` command
```

Comment:
What you write is correct, when I did it I just used the https://github.com/rhinstaller/anaconda/pull/6597.patch, it's basically creating the patch file from the PR for you.

you can do the same for isolated commits: https://github.com/rhinstaller/anaconda/commit/$SHA.patch

Download the file, copy it in the repository root, and use the URL in the Patch: line

Reply:
I'd like to keep this in place as in some cases git operations might be necessary to get patches that apply to the version in the distgit tarball. But I'll add a note about using the GitHub URL tricks to get patch files for the simple cases. :)

Reply:
BTW, I will skip suggesting including the URL - given that you still need to store and `git add` the fine anyway, I think that would be confusing. Keeping just the guidance to work one file level is IMHO more robust there.